CMakeLists.txt

```diff
@@ -32,13 +32,8 @@ set(ZEROEQ_MAINTAINER "Human Brain Project <HBPVis@googlegroups.com>")
 set(ZEROEQ_LICENSE LGPL)
 set(ZEROEQ_DEPENDENT_LIBRARIES Servus)
 
-# enforce cloning our own version of cppnetlib and skip finding a potentially
-# installed system package
-git_external(${COMMON_SOURCE_DIR}/cppnetlib
-  https://github.com/BlueBrain/cpp-netlib 0.11-bbp)
-add_subproject(cppnetlib)
-
 common_find_package(Boost COMPONENTS unit_test_framework)
+add_subproject(cppnetlib ${PROJECT_SOURCE_DIR}/cppnetlib) # from .gitmodules
```

Comment:
I'd still keep it separate at the top as before, this line looks out of place to me.